contributing_to_docs/contributing.adoc

```diff
@@ -14,22 +15,18 @@ There are a few different ways you can contribute to OpenShift documentation:
 
 // * Submit comments at the bottom of each topic (still awaiting implementation)
 * Email the OpenShift documentation team openshift-docs@redhat.com
-* https://github.com/openshift/openshift-docs/issues/new[Create an issue in GitHub]
-* Submit a PR. You can clone the repository, make your changes, and submit a PR.
-This option is best if you have substantial changes. If you open a PR, be sure
-that all of its contents are related and apply to the same versions.
+* Create a link:https://github.com/openshift/openshift-docs/issues/new[GitHub], link:https://issues.redhat.com/secure/CreateIssue!default.jspa[Jira], or link:https://bugzilla.redhat.com/enter_bug.cgi?product=OpenShift%20Container%20Platform[Bugzilla] documentation ticket.
+* Submit a pull request (PR). You can create a local clone of your own fork of the link:https://github.com/openshift/openshift-docs[openshift-docs repository], make your changes, and submit a PR. This option is best if you have substantial changes. If you open a PR, be sure that all of its contents are related and apply to the same versions.
 
 *What happens when you submit a PR?*
 
-The
-https://github.com/orgs/openshift/teams/team-documentation[documentation team]
-reviews the PR and arranges further review by the development and quality
-assurance teams, as required.
-If the PR requires changes, updates, or corrections, we will let you know
-in the PR. We might request that you make the changes, or let you know that we
-incorporated your content in a different PR. When the PR has been reviewed, all
-updates are complete, and all commits are squashed, we'll merge your PR and
-apply it to the valid versions.
+The following diagram outlines the OpenShift documentation contribution process:
+
+image::osdocs-contribution-workflow.png[OpenShift documentation contribution workflow]
```

Comment:
I have requested a further update to the diagram to change "(Jira, BZ, GitHub)" to "(Jira, BZ, GitHub, email)" so that the diagram is consistent with this section in the docs.

Reply:
Very unimportant but in the image there's a mix of "GitHub" and "Github" capitalization.

Reply:
Good catch! I have subsequently requested and update to the diagram to resolve this.

Reply:
An updated diagram has been applied. Both of these points are resolved.